orca-front50/src/test/groovy/com/netflix/spinnaker/orca/front50/DependentPipelineStarterSpec.groovy

```diff
@@ -81,7 +81,8 @@ class DependentPipelineStarterSpec extends Specification {
       triggeredPipelineConfig,
       null /*user*/,
       parentPipeline, [:],
-      null
+      null,
+      true
```

Comment:
Just blanket updated these tests.

If there's some agreement that we should fix this, I'll add a set of tests to validate the `false` path.